theme/less/main.less

```diff
@@ -1,23 +1,26 @@
 @import "_variables.less";
 @import "_navigation.less";
+@import "_header.less";
+@import "_grid.less";
 
 body
 {
-  margin: 0px;
-  font-family: sans-serif;
+  font-family: Roboto, sans-serif;
+  margin: 0;
 }
 
-header
-{
-  background-color: @primary;
-  padding: 30px;
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 500;
+  src: local('Roboto Medium'), local('Roboto-Medium'), url(/fonts/roboto-500.woff2) format('woff2');
```

Comment:
What current font stands for - Latin, Latin + latin-extended? Let's have consistent naming and also folder structure, what about:
```
fonts
 roboto
   500
      cyrillic.woff2
```

Reply:
It's Latin. Done.